src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -1537,6 +1556,16 @@ bool InterpreterCreateQuery::doCreateTable(ASTCreateQuery & create,
         }
     }
 
+    if (UInt64 max_table = getContext()->getGlobalContext()->getServerSettings().max_table_num_to_throw; max_table > 0)
+    {
+        UInt64 table_count = CurrentMetrics::get(CurrentMetrics::AttachedTable);
+        if (table_count >= max_table)
+            throw Exception(ErrorCodes::TOO_MANY_TABLES,
+                            "Too many tables in the system. Current is {}, limit is {}. "
```

Comment:
How about using one sentence:
`Too many tables in ClickHouse. The limit (setting 'max_table_num_to_throw') is set to {}`

Same for databases.

Besides, do write period at the end.